src/main/java/controllers/LoginController.java

```diff
@@ -111,11 +125,18 @@ private boolean runWelcomeAction(Integer welcomeAction) {
         }
     }
 
+    /**
+     * Run logout logic
+     */
```

Comment:
I think instead of sout and printing directly, we should use inOut.setOutput right?

Reply:
Yes you're right! Must've missed that...